diff --git a/examples/companion_radio/main.cpp b/examples/companion_radio/main.cpp
index 89f0e6cb..0fc2a6bc 100644
--- a/examples/companion_radio/main.cpp
+++ b/examples/companion_radio/main.cpp
@@ -171,7 +171,7 @@ void setup() {
     #endif
   );
 #elif defined(ESP32)
-  SPIFFS.begin(true);
+  SPIFFS.begin(true, "/spiffs", 10, "mc_fs");
   store.begin();
   the_mesh.begin(
     #ifdef DISPLAY_CLASS
